charts/kaito/workspace/values.yaml

```diff
@@ -16,7 +16,7 @@ securityContext:
     drop:
       - "ALL"
 featureGates:
-  vLLM: "true"
+  vLLM: true
```

Comment:
Could you use the `gatewayAPIInferenceExtension` feature gate from https://github.com/kaito-project/kaito/pull/1252/files#diff-b591626b039f6bb7fe8aebf16f6f940e240dbc580024668ca2b05732cbba5081R36 instead? It can default to false first.